products/osconfig/terraform.yaml

```diff
@@ -58,7 +58,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           function: 'validation.IntBetween(-1,31)'
       recurringSchedule.frequency: !ruby/object:Overrides::Terraform::PropertyOverride
         exclude: true
-      rollout.disruptionBudget.percentage: !ruby/object:Overrides::Terraform::PropertyOverride
+      rollout.disruptionBudget.percent: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
(if you do the `api_name` change above, this can be reverted back)

Reply:
Done and done